dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/ExtendedDocumentReference.kt

```diff
@@ -0,0 +1,15 @@
+package org.dataland.datalandbackend.model
+
+import org.dataland.datalandbackend.interfaces.ExtendedDocumentReferenceInterface
+
+/**
+ * --- API model ---
+ * A reference to a page in a company report
+ */
+data class ExtendedDocumentReference(
+    override val page: Long? = null,
+    override val tagName: String? = null,
+    override val fileName: String? = null,
+    // TODO check that the @field:JsonProperty(required = true) can be removed here without problems
```

Comment:
A data source with all fields nullable does not make sense to me. The data source should be nullable in some cases but providing a datasource that is empty seems pointless.

Reply:
Not changed as discussed in a call

Reply:
I had a quick look at this. If i change this to be required then it is not possible to upload datapoints without data sources and i think that we don't want.

Reply:
Changed